app/src/utils/utils.ts

```diff
@@ -25,3 +25,20 @@ export function isValidUrl(val: string) {
 export function isValidAddress(val: string) {
   return isAddress(val);
 }
+
+// Expects a unix timestamp and will return a human readable message of how far in the past/future it is
+export function daysAgo(val = 0) {
+  // use a formatter to establish "in 10 days" vs "10 days ago"
+  const formatter = new Intl.RelativeTimeFormat();
+
+  // days ago
+  const deltaDays = (val * 1000 - Date.now()) / (1000 * 3600 * 24);
+
+  // Format days to string (rember to round off deltaDays)
+  return formatter.format(Math.round(deltaDays), 'days');
+}
+
+// convert a unix ts to a toLocaleString
+export const unixToLocaleString = (time: BigNumber) => {
+  return new Date(time.toNumber() * 1000).toLocaleString();
+};
```

Comment:
May as well make this a bit more generic. Also might want to change this from `const` to `function` to be consistent with the rest of the file. (or we can change them all to arrow functions if you prefer, for this utils file I don't think it matters much)

```suggestion
export function unixToLocaleString(time: BigNumberish) {
  return new Date(BigNumer.from(time).toNumber() * 1000).toLocaleString();
};
```

Reply:
Resolved in b35f417